### DISCUSSION TITLE
How to show prefix (if there is one set) right before note's title/name

### DISCUSSION THREAD

**Discussion by MotikaCraft-TheJoshua** [TriliumNext/Trilium#4360] [Q&A]
Question is in the Discussion title

![image](https://github.com/zadam/trilium/assets/56804505/b74ed839-33f3-42a6-9557-329fabf6c5d9)

Thank you in advance!

**Comment by zadam** [comment]
This isn't available ... the idea is that prefix is associated with a branch which is a tree only property, while the right pane is only about the note itself.

But to be honest I'm not completely sure if this separation is intuitive/useful. I'll think about this a bit more.
